store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/util/MergeSchema.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2017-2022 Crown Copyright
```

Comment:
Work started on this in 2017?

Reply:
no.

Reply:
No but I think it was duplicated from an earlier class.